sdk/core/core-https/src/policies/exponentialRetryPolicy.ts

```diff
@@ -17,6 +17,13 @@ const DEFAULT_CLIENT_RETRY_COUNT = 10;
 const DEFAULT_CLIENT_RETRY_INTERVAL = 1000;
 const DEFAULT_CLIENT_MAX_RETRY_INTERVAL = 1000 * 64;
 
+const retryHttpStatusCodes: number[] = [
+  // Client error responses
+  408, // Request timeout
+  425, // Too early
+  429 // Too many requests
```

Comment:
We should let throttlingRetryPolicy handle this instead, per Jeremy's comment.

Or was the service not providing a Retry-After header?

Reply:
Good point, I need to put more time testing this! I'll come back with more clarity after the identity releases this month.